crates/gen-host-wasmtime-rust/tests/runtime.rs

```diff
@@ -11,43 +14,33 @@ fn default_config() -> Result<Config> {
     let mut config = Config::new();
     config.cache_config_load_default()?;
     config.wasm_backtrace_details(wasmtime::WasmBacktraceDetails::Enable);
+    config.wasm_component_model(true);
     Ok(config)
 }
 
-fn default_wasi() -> wasmtime_wasi::WasiCtx {
-    wasmtime_wasi::sync::WasiCtxBuilder::new()
-        .inherit_stdio()
-        .build()
-}
-
-struct Context<I, E> {
-    wasi: wasmtime_wasi::WasiCtx,
+struct Context<I> {
     imports: I,
-    exports: E,
 }
 
-fn instantiate<I: Default, E: Default, T>(
+fn instantiate<I: Default, T>(
     wasm: &str,
-    add_imports: impl FnOnce(&mut Linker<Context<I, E>>) -> Result<()>,
+    add_imports: impl FnOnce(&mut Linker<Context<I>>) -> Result<()>,
     mk_exports: impl FnOnce(
-        &mut Store<Context<I, E>>,
-        &Module,
-        &mut Linker<Context<I, E>>,
+        &mut Store<Context<I>>,
+        &Component,
+        &mut Linker<Context<I>>,
     ) -> Result<(T, Instance)>,
-) -> Result<(T, Store<Context<I, E>>)> {
+) -> Result<(T, Store<Context<I>>)> {
     let engine = Engine::new(&default_config()?)?;
-    let module = Module::from_file(&engine, wasm)?;
+    let module = Component::from_file(&engine, wasm)?;
 
     let mut linker = Linker::new(&engine);
     add_imports(&mut linker)?;
-    wasmtime_wasi::add_to_linker(&mut linker, |cx| &mut cx.wasi)?;
```

Comment:
This is removing wasi entirely for tests, which I think means that tests can no longer have debug prints within them, right? Would you be ok updating the `preview1.wasm` adapter used to delegates writes on fd 1 to a simple `log: func(s: &str)` function?

Reply:
I can go add that in, but maybe as a follow on PR. I don't think tests currently have debug prints because if they did, they'd trap in the current implementation.

Reply:
https://github.com/bytecodealliance/wit-bindgen/pull/371